### PR TITLE
Add workflow description

### PR DESCRIPTION
**Pull request by dreampulse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,3 +60,44 @@ The following folders are unit tested and should have 100% test coverage:
...
+- The code is concise, expressive and readable
+- The code is functional and doesn’t use class syntax
+
+### React components are
+- Stateless and use pure rendering functions
+- Are documented and tested in storybook
+- Typed-checked
+- The Sass is strictly using the BEM-Convention
+- For each react component exists exactly one Sass-File
+- The is no container styled
```
Comment (jhnns) [inline README.md]: Typo: There is...

File: README.md [hunk 2]
```diff
@@ -60,3 +60,44 @@ The following folders are unit tested and should have 100% test coverage:
...
+- There are no 'hacks' or shortcuts refactoring is preferred
+
+###	Testability
+- All exported functions are testable
+- Default exports are avoided
+- The code is concise, expressive and readable
+- The code is functional and doesn’t use class syntax
+
+### React components are
+- Stateless and use pure rendering functions
```
Comment (jhnns) [inline README.md]: Split into two bullet points:
- are stateless
- use pure rendering functions

File: README.md [hunk 3]
```diff
@@ -60,3 +60,44 @@ The following folders are unit tested and should have 100% test coverage:
...
+- Are documented and tested in storybook
+- Typed-checked
+- The Sass is strictly using the BEM-Convention
+- For each react component exists exactly one Sass-File
```
Comment (jhnns) [inline README.md]: For each React component, there is only one Sass file
? 😁

File: README.md [hunk 4]
```diff
@@ -60,3 +60,44 @@ The following folders are unit tested and should have 100% test coverage:
...
+1. The implementer of a bugfix / new feature creates a pull request (PR) in github.
+1. The implementer adds a reviewer to the pull request when the implementation is done.
+  - The reviewer is selected by the implementer.
+  - The reviewer is added to the list of **reviewers** and **assignees**.
+  - The label `needs review` is added to the pull request.
+1. The reviewer reviews the PR and adds comments for each known issue.
+  - The reviewer has to update the label to `needs fixing` and to change the **assignees** to the implementer.
+  - If there are no issues left the reviewer **and only him** has to merge the PR to the `master`.
+1. If there are still open issues the implementer **and only him** has to fix them.
+  - Fixed issues has to be marked by using the :+1:-emoji.
```
Comment (jhnns) [inline README.md]: Typo: Fixed issues have to be...

Comment (jhnns) [inline README.md]: Why does the issue have to be marked with that particular emoji?

Comment (dreampulse) [reply README.md]: To indicate that the issue in the comment has been fixed.
The emoji has a nice advantage: you don't get notified by an email.

**Pull request merged**